src/plugins/mer/meremulatormodedialog.cpp

```diff
@@ -58,7 +62,7 @@ MerEmulatorModeDialog::MerEmulatorModeDialog(QObject *parent)
       m_kit(0)
 {
     m_action->setEnabled(false);
-    m_action->setText(tr("&Emulator mode..."));
+    m_action->setText(QString("&%1").arg(MER_EMULATOR_MODE_ACTION_NAME));
```

Comment:
That's not going to work. Add the ampersand where MER_EMULATOR_MODE_ACTION_NAME is defined and use `setText(tr(MER_EMULATOR_MODE_ACTION_NAME))` here.

Reply:
Fixed.